student.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission, name = 'Unknown')
+    @classroom = classroom
+    super(
+      @age = age,
+      @classroom = classroom,
+      @name = name,
+      @parent_permission = parent_permission
+  )
```

Comment:
- [ ] You don't need to send all variables to the parent class. Kindly fix this issue as shown below 👍

```
super(
  age,
  parent_permission,
  name
)
```